File: 3rdSemester/Object Oriented Programming Lab/Lab 1/Activity1_4_2.cpp
#include <iostream>
using namespace std;

class Point
{

private:
    int xCordinate, yCordinate;

public:
    Point()
    {
        xCordinate = 0;
        yCordinate = 0;
    }

    Point(int x, int y)
    {
        xCordinate = x;
        yCordinate = y;
    }

    int getX()
    {
        return xCordinate;
    }

    int getY()
    {
        return yCordinate;
    }
    void setX(int x)
    {
        xCordinate = x;
    }

    void setY(int y)
    {
        yCordinate = y;
    }

    void display()
    {
        cout << "( " <<xCordinate << " , "<< yCordinate<<")" << endl;
    }
};

int main()
{
    Point p1;
    p1.display(); //Call display() function to show initial value of p1
    p1.setX(2); p1.setY(3);
    p1.display(); //Call display() function to show updated value of p1

    Point p2(5,2);
    p2.display(); //Call display() function to show initial value of p2

    p2.setX(6); p2.setY(3);
    p2.display(); //Call display() function to show updated value of p2

    return 0;

}

File: 3rdSemester/Object Oriented Programming Lab/Lab 1/Python/Activity1_4_2.py
from mimetypes import init


class Point:
    def __init__(self):
        self.x = 0
        self.y = 0

    def __init__(self,a,b):
        self.x = a
        self.y = b

    def setX(self,a):
        self.x = a

    def setY(self,b):
        self.y = b

    def getX(self):
       return self.x

    def getY(self):
        return self.y

    def show(self):
        print("( "+str(self.x)+" , "+str(self.y)+" )")

P1=Point(0,0)
P2=Point(2,4)

P1.setX(3)
P1.setY(8)
print("Point 1: "+str(P1.getX())+" "+str(P1.getY()))
print("Point 2: "+str(P2.getX())+" "+str(P2.getY()))